vello_shaders/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "vello_shaders"
+version = "0.1.0"
+edition = "2021"
+
+[features]
+default = ["compile", "wgsl", "msl"]
+compile = ["naga"]
+wgsl = []
+msl = []
+
+[dependencies]
+naga = { git = "https://github.com/gfx-rs/naga", features = ["wgsl-in", "msl-out", "validate"], optional = true }
```

Comment:
This needs to pin a git hash, as it's currently failing (missing `msl::Options::zero_initialize_workgroup_memeory`)